server/src/diagnostics.ts

```diff
@@ -324,6 +346,16 @@ export class Diagnostics {
     return Range.create(textDocument.positionAt(attributeValueStart), textDocument.positionAt(attributeValueEnd))
   }
 
+  private createParseErrorDiagnosticFor(identifier: string, textDocument: TextDocument, range: Range) {
+    this.pushDiagnostic(
+      `There was an error parsing the "${identifier}" Stimulus controller. Please check the controller for syntax errors.`,
```

Comment:
I was wondering if it would make sense to actually show the error message of the parseError here in the diagnostic as well. is there a reason why you might have left it out?

Reply:
I think `SyntaxError: Unexpected token (23:2)` is useful if the location is actually accurate.

Reply:
Maybe it would make sense if we add the whole exception object to the controller definition in that case, so we can everything we need